VALUES.md

```diff
@@ -0,0 +1,19 @@
+We want to make sure every member has a shared understanding of the goals and values we hold as a team:
+
+- Optimize for the **overall project**, not your own area or feature
+  - A shortcut for one individual can mean a lot of extra work or disruption for the rest of the team.
+- Our repos should always be in release shape: **Always Green**
+  - This lets us move faster in the mid and long term
+  - This implies investments in build/test infrastructure to have fast, reliable tests to ensure that we can release at any time.
+  - Extra discipline may require more work by individuals to keep the build in good state, but less work overall for the team.
+- Be **specific**, **respectful** and **courteous**
+  - Disagreements are welcome and encouraged, but don't use broad generalizations, exaggerations, or judgement words that can be taken personally. Consider other people’s perspective (including the wide range of applicability of Istio). Empathize with our users. Focus on the specific issue at hand, and remember that we all care about the project, first and foremost.
+  - If you feel upset - disengage, then write down what bothers you in an email to yourself - don’t send it to anyone. If you feel the same way the next morning, find a constructive way to address the issue or raise it with a friend or a lead.
```

Comment:
I think this bullet is somewhat preachy and too prescriptive. We're not talking to 5 year olds here. I would just remove it.